x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/protections/malware.tsx

```diff
@@ -189,14 +191,17 @@ export const MalwareProtections = React.memo(() => {
             />
           </h6>
         </EuiTitle>
+        <EuiText color="subdued" size="xs">
+          <i>{`Endpoint versions ${popupVersionsMap.get('malware')}`}</i>
```

Comment:
I think this copy should refer to `Agent versions` over Endpoint.  Users in Fleet will upgrade Agents as opposed to just Endpoints.

What are your thoughts @bfishel @caitlinbetz